keps/prod-readiness/sig-node/3545.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 3545
 alpha:
-  approver: "@johnbelamaric"
\ No newline at end of file
+  approver: "@johnbelamaric"
+beta:
+  approver: "@johnbelamaric"
```

Comment:
correct, (AFAIK) the protocol is to keep the previous reviewer and let PRR team rebalance if needed